kafka/channel/README.md

```diff
@@ -125,14 +87,46 @@ kubectl get deployment -n knative-eventing kafka-webhook
 The Kafka Config Map is used to configure the `bootstrapServers` of your Apache
 Kafka installation:
 
-- **cluster-scoped**:
-
 ```shell
 kubectl get configmap -n knative-eventing config-kafka
 ```
 
-- **namespace-scoped**:
+### Namespace Dispatchers
 
-```shell
-kubectl get configmap   config-kafka
+By default events are received and dispatched by a single cluster-scoped dispatcher components. You can also specify whether events should be received and dispatched by the dispatcher in the same namespace as the channel definition by adding the
+`eventing.knative.dev/scope: namespace` annotation.
+
+First, you need to create the configMap `config-kafka` in the same namespace as the KafkaChannel.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-kafka
+  namespace: <YOUR_NAMESPACE>
+data:
+  # Broker URL. Replace this with the URLs for your kafka cluster,
+  # which is in the format of my-cluster-kafka-bootstrap.my-kafka-namespace:9092.
+  bootstrapServers: REPLACE_WITH_CLUSTER_URL
+```
+
+Then create a KafkaChannel:
+
+```yaml
+apiVersion: messaging.knative.dev/v1alpha1
+kind: KafkaChannel
+metadata:
+  name: my-kafka-channel
+  namespace: <YOUR_NAMESPACE>
+  annotations:
+    eventing.knative.dev/scope: namespace
+spec:
+  numPartitions: 1
+  replicationFactor: 3
```

Comment:
This requires 3 NOTES

our doc does only install small cluster w/ one node 


the same config error is on the snippet for the cluster wide disptacher 

@lionelvillard can you chagen ? 

`numPartitions`  can be larger than the number of nodes, but not the `replication`